### ISSUE TITLE
External lua libraries

### ISSUE THREAD

**Issue by ht250763** [inmation/library#2]
Hello,
is there a way to embed other external lua libraries into inmation or some kind of a package manager?

Actually we tried to establish an ODBC connection to a Honeywell DCS using the lua-sql library but failed because the needed connection string syntax is not supported by this library. 

[https://github.com/keplerproject/luasql/issues/96](url)

There is a good chance for getting a connection using ADO object. However for this we would need LuaCom library [http://webserver2.tecgraf.puc-rio.br/~rcerq/luacom/](url).

Is there a way to get this library in?

Regards

ht

**Comment by Longmountain** [comment]
Yes, you can store **pure** Lua libraries into inmation on e.g. folder like objects.

There is an ODBC library available on inmation-ESI. Do you have access to this repository? Otherwise please create a ticket on our support page so that we can help you.
